src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java

```diff
@@ -40,6 +43,14 @@ public TopicResource(TopicService topicService) {
         return ResponseEntity.ok(GetAllTopicsResponse.fromTopicWithFilteredChildren(parentTopics, filteredChildTopics));
     }
 
+    @GetMapping(value = "/case/foitopiclist", produces = APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<Set<TopicDto>> getAllFOITopics() {
+        log.info("requesting all topics from FOI Topics list");
+        ParentTopic parentTopic = parentTopicRepository.findByDisplayName("FOI Topics");
```

Comment:
I think from a single responsibility point of view and ease of readability I think this line should be moved into `topicsService` under a new method. I would add that I think the processing fo the topics in the stream below should also be in there also.

Reply:
Ugh! Ok... i'm not keen but I can't argue the facts. Maybe we should raise it in retro though on a clean up policy going forward. Cheers